charts/camunda-platform/Chart.yaml

```diff
@@ -55,6 +49,10 @@ dependencies:
     repository: "https://helm.elastic.co"
     version: 7.17.1
     condition: "elasticsearch.enabled"
+  - name: postgresql
+    repository: https://charts.bitnami.com/bitnami
+    version: 11.x.x
+    condition: "postgresql.enabled"
```

Comment:
FYI: Unfortunately, Helm's support for enabling subcharts based on a condition is quite limited. I would have needed something like `webModeler.enabled && postgresql.enabled`, because the PostgreSQL dependency is optional and you can also use Web Modeler with an external database.
So, I decided to disable the PostgreSQL chart by default (see `values.yaml`).

Reply:
yeah I think that's the best that could be done now.